src/m365/spo/commands/folder/folder-get.ts

```diff
@@ -87,19 +93,18 @@ class SpoFolderGetCommand extends SpoCommand {
     if (this.verbose) {
       logger.logToStderr(`Retrieving folder from site ${args.options.webUrl}...`);
     }
-
-    let requestUrl: string = '';
-
+    let requestUrl: string = `${args.options.webUrl}/_api/web`;
     if (args.options.id) {
-      requestUrl = `${args.options.webUrl}/_api/web/GetFolderById('${formatting.encodeQueryParameter(args.options.id)}')`;
+      requestUrl += `/GetFolderById('${formatting.encodeQueryParameter(args.options.id)}')`;
     }
     else if (args.options.url) {
       const serverRelativeUrl: string = urlUtil.getServerRelativePath(args.options.webUrl, args.options.url);
-      requestUrl = `${args.options.webUrl}/_api/web/GetFolderByServerRelativeUrl('${formatting.encodeQueryParameter(serverRelativeUrl)}')`;
+      requestUrl += `/GetFolderByServerRelativeUrl('${formatting.encodeQueryParameter(serverRelativeUrl)}')`;
     }
 
+    const propertiesPermissions = args.options.withPermissions ? `?$expand=ListItemAllFields/HasUniqueRoleAssignments,ListItemAllFields/RoleAssignments/Member,ListItemAllFields/RoleAssignments/RoleDefinitionBindings` : ``;
     const requestOptions: any = {
-      url: requestUrl,
+      url: requestUrl + propertiesPermissions,
       headers: {
```

Comment:
lets refactor to something like 
```typescript
if(args.options.withPermissions) {
      requestUrl += `?$expand=ListItemAllFields/HasUniqueRoleAssignments,ListItemAllFields/RoleAssignments/Member,ListItemAllFields/RoleAssignments/RoleDefinitionBindings`;
    }
    
    const requestOptions: any = {
      url: requestUrl,
      headers: {
        'accept': 'application/json;odata=nometadata'
      },
      responseType: 'json'
    };
```

I think it will cleaner 🤔